lib/modules/manager/npm/post-update/yarn.spec.ts

```diff
@@ -409,6 +409,47 @@ describe('modules/manager/npm/post-update/yarn', () => {
     expect(res.lockFile).toBe('package-lock-contents');
   });
 
+  it('falls back to Yarn v1 for undefined constraint', async () => {
```

Comment:
Apparently we default to Yarn v2 due to this: https://github.com/renovatebot/renovate/blob/66db519d3707e606e5849e521b58dec0197093e4/lib/modules/manager/npm/extract/yarn.ts#L102-L116